ivy/functional/frontends/numpy/ndarray/ndarray.py

```diff
@@ -500,6 +500,9 @@ def __ixor__(self, value, /):
     def __imod__(self, value, /):
         return np_frontend.mod(self, value, out=self)
 
+    def __invert__(self, out=None, /):
+        return ivy.bitwise_invert(self.ivy_array, out=out)
```

Comment:
```suggestion
    def __invert__(self, /):
        return ivy.bitwise_invert(self.ivy_array)
```

Reply:
We don't need any out argument here - there is no out arg in the numpy function https://numpy.org/doc/stable/reference/generated/numpy.ndarray.__invert__.html